### PR TITLE
UICONSET-168 Split consortium manager's navigation links into sections

### PR DESCRIPTION
**Pull request by usavkov-epam**
## Purpose

https://folio-org.atlassian.net/browse/UICONSET-168

---
Linter is broken and should be fixed.

## Approach

Use `Map` structure instead of a list to store and identify modules by sections in the consortium manager. Resolve accordingly related modules in the `<ConsortiumManager />`.

Implement toggle for the consortium's navigation pane and keep the interface in the consortium manager context.

## Screenshots

https://github.com/user-attachments/assets/70814e1a-f800-45bd-a83c-f38212f53722

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [x] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [x] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/components/ConsortiumManagerNavigationPaneToggle/ConsortiumManagerNavigationPaneToggle.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import { PaneHeaderIconButton } from '@folio/stripes/components';
+
+import { useCallback } from 'react';
```
Comment (BogdanDenis) [inline src/components/ConsortiumManagerNavigationPaneToggle/ConsortiumManagerNavigationPaneToggle.js:3]: Please re-order imports so the are in this order:
1. Third party libraries
2. Stripes imports
3. Local module imports

Please also check other files in this PR

**Review by BogdanDenis — CHANGES_REQUESTED** [review]

File: src/components/ConsortiumManagerNavigationPaneToggle/ConsortiumManagerNavigationPaneToggle.js (hunk 1 above)
Comment (usavkov-epam) [reply src/components/ConsortiumManagerNavigationPaneToggle/ConsortiumManagerNavigationPaneToggle.js]: Good catch, thanks! Updated

File: package.json [hunk 2]
```diff
@@ -27,9 +27,12 @@
     "@folio/stripes": "^9.0.0",
     "@folio/stripes-cli": "^3.0.0",
     "@formatjs/cli": "^6.1.3",
+    "babel-jest": "^26.3.0",
     "core-js": "^3.6.1",
     "eslint": "^7.32.0",
     "eslint-plugin-filenames": "^1.3.2",
+    "eslint-plugin-import": "^2.22.1",
```
Comment (Terala-Priyanka) [inline package.json:34]: I think this package is available from "@folio/eslint-config-stripes". Do you need to explicitly add it?

File: package.json [hunk 3]
```diff
@@ -27,9 +27,12 @@
     "@folio/stripes": "^9.0.0",
     "@folio/stripes-cli": "^3.0.0",
     "@formatjs/cli": "^6.1.3",
+    "babel-jest": "^26.3.0",
```
Comment (Terala-Priyanka) [inline package.json:30]: I think this package is available from "@folio/jest-config-stripes". Do you need to explicitly add it?

**Review by BogdanDenis — APPROVED** [review]

File: package.json (hunk 2 above)
Comment (usavkov-epam) [reply package.json]: Updated

File: package.json (hunk 3 above)
Comment (usavkov-epam) [reply package.json]: Updated

**Review by Terala-Priyanka — APPROVED** [review]

**Pull request merged**